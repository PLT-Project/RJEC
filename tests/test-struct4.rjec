struct foo {
    x bool;
    y int;
    z bool;
}

func main() {
    var i struct foo;
    i.x = true;
    i.y = 9;
    i.z = true;
    
    printb(i.x);
    printi(i.y);
    printb(i.z);

}