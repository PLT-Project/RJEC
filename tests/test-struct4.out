true
9
true